docs/changelog.rst

```diff
@@ -1,6 +1,57 @@
 Release Notes
 =============
 
+1.13
+----
+
+2023-06-15
+
+- Support for aliases (:ghpull:`367`), to provide shorter, more meaningful names
+  for specific sources & keys, and support for loading a default set of aliases
+  for the proposal when using :func:`~.open_run` (:ghpull:`398`). See
+  :ref:`using-aliases` for more information.
+- New APIs for multi-module detector data to work more like regular sources and
+  keys, e.g. ``agipd['image.data'].ndarray()`` (:ghpull:`337`). These changes
+  also change how Dask arrays are created for multi-module detector data,
+  hopefully making them more efficient for typical use cases.
+- New method :meth:`~.DataCollection.plot_missing_data` to show where sources
+  are missing data for some trains (:ghpull:`402`).
+- Merging data with :meth:`~.union` now applies the same train IDs to all
+  included sources, whereas previously sources could have different train IDs
+  selected (:ghpull:`416`).
+- A new property ``run[src].device_class`` exposes the Karabo device class name
+  for control sources (:ghpull:`390`).
+- :class:`.JUNGFRAU` now accepts a ``first_modno`` for detectors where the first
+  module is named with e.g. ``JNGFR03`` (:ghpull:`379`).
+- ``run[src].is_control`` and ``.is_instrument`` properties (:ghpull:`403`).
+- :class:`.SourceData` objects now have ``.data_counts()``,
+  ``.drop_empty_trains()`` and ``.split_trains()`` methods like :class:`.KeyData`
+  (:ghpull:`404`, :ghpull:`405`, :ghpull:`407`).
+- New method ``SourceData.one_key()`` to quickly find an arbitrary key for a
+  source.
+- :meth:`~.DataCollection.select` now accepts a ``require_any=True`` parameter
+  to filter trains where any of the selected sources & keys have data,
```

Comment:
Ah, last minute comment after it spun up in my head:

This may be confusing, as no sources is also any source... so maybe add "where any but at least one of the selected sources & keys have data"

Reply:
I would say no sources is not any source, but I'll tweak the wording anyway to ensure it's clear.